django/contrib/admin/views/autocomplete.py

```diff
@@ -24,11 +26,29 @@ def get(self, request, *args, **kwargs):
             return JsonResponse({'error': '403 Forbidden'}, status=403)
 
         self.term = request.GET.get('term', '')
+        app_label = request.GET.get('app_label')
+        model_name = request.GET.get('model_name')
+        field_name = request.GET.get('field_name')
+
+        model = apps.get_model(app_label, model_name)
+
+        try:
+            self.field = model._meta.get_field(field_name)
```

Comment:
If I understood the above code correctly, then `self.field` is on the source model, whereas `self.model_admin` points to the target admin, I think we should really rename those to avoid confusion.